libs/dart/didcomm/atm_client/lib/src/models/request_bodies/base_deploy_instance_request.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'base_deploy_instance_request.g.dart';
+
+/// Request for deploying an instance.
+/// 
+/// Contains all fields required for deployment across all instance types.
+@JsonSerializable(includeIfNull: false, explicitToJson: true)
+class BaseDeployInstanceRequest {
+  /// Administrator DIDs for the instance.
+  final String? administratorDids;
```

Comment:
Im somewhat out of context for this one, but just double checking: 
are not these meant to be snake case? Sometihng like `administrator_dids`, .etc?